docs/running/tardis_widgets/using_widgets.rst

```diff
@@ -0,0 +1,105 @@
+********************
+Using TARDIS Widgets
+********************
+
+This page describes what each TARDIS Widget has to offer and how you can make
+the best use of it. If you're looking for the code to generate widgets, head
+over to `Generating TARDIS Widgets <generating_widgets>`_ section and see the
+notebook in action.
+
+Currently, TARDIS supports the following widgets:
+
+Shell Info Widget
+#################
+
+This widget allows you to explore the chemical abundances in different shells
+of a Supernova ejecta.
+
+.. image:: images/shell-info-widget-demo.gif
+    :alt: Demo of Shell Info Widget
+
+It consists of four interlinked tables - clicking on any row in a table,
+populates data in the table(s) right to it. All these 4 tables, in order, has
+following information:
+
+1. **Shells Data** - Radiative temperature and Dilution Factor (W) of each shell
+(computational partitions) of a Supernova ejecta. Shell numbers in ascending 
+order represent inner to outer i.e. innermost shell is numbered as 1 and 
+outermost shell is numbered as the last number (say 20).
+
+2. **Element Abundances** - Fractional abundance of each element present in the
+selected shell.
+
+3. **Ion Abundances** - Fractional abundance of each ion (species) of the
+selected element present in the selected shell. 
+
+4. **Level Abundances** - Fractional abundance of each level of the selected
+ion and element in the selected shell.
+
+.. any more info about ion no.s (0 means no charge and hence I) and level no.s
+
+Line Info Widget
+################
+
+This widget lets you explore line interactions present in the spectrum of a
+simulation model.
+
+.. image:: images/shell-info-widget-demo.gif
+    :alt: Demo of Line Info Widget
+
+You can select any wavelength range in the spectrum interactively to display a 
+table of species abundances (fraction of packets interacting). Using 
```

Comment:
"display a table giving the fraction of packets that experienced their last interaction with each element."